### ISSUE TITLE
Filters dropdown: grey out "Apply filters" button in map mode since it does not have effects there

### ISSUE THREAD

**Issue by rowasc** [ushahidi/platform#2182]
### Expected behaviour
I see an Apply Filters button in the map but it's greyed out.
### Actual behaviour
I see an Apply Filters button in the map, where it has no effect since we are applying them as we go.
### Steps to reproduce the behaviour/error
Go to the map and open the filters dropdown.

##### Where
- [ ] [Local setup with Vagrant ](https://www.ushahidi.com/support/install-ushahidi#installing-for-development)
- [ ] [Local setup from platform-release ](https://www.ushahidi.com/support/install-ushahidi#installing-the-latest-release)
- [ ] Ushahidi.io / SaaS solution 
- [x] Ushahidi's QA environment
- [x] Other (explain): sandbox

**Comment by rowasc** [comment]
Creating this from a conversation here https://github.com/ushahidi/platform-client/pull/838

**Comment by rowasc** [comment]
@aoduor @jshorland is this reasonable? I think it'd make sense to hide it there since it only makes users wonder "Why is this even here". And at the same time it makes it a bit more obvious in the data mode that we need to apply them, since it's there. 
TLDR: Super easy to do, and I think it adds value. LMK

**Comment by justinscherer** [comment]
I'd rather "grey out" the "apply filters" button.

**Comment by rowasc** [comment]
ah, super. I think that makes sense 👍  Thanks

**Comment by justinscherer** [comment]
Low priority right now, because there are a bunch of UI bugs in the filter pane right now which are much more important than this one.
